src/components/SalaryBar/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Progress } from 'reactstrap';
+import { Offer, PayType } from '../../api/types/offer';
+import { JobType } from '../../api/types/position';
+
+interface SalaryBarProps {
+  offers: Offer[];
+  display?: JobType;
+}
+
+export const SalaryBar: React.FunctionComponent<SalaryBarProps> = ({
+  offers,
+  display
+}) => {
+  const salaries = offers
+    .filter(
+      offer => offer.position.job_type === display || display === undefined
+    )
+    .map(offer => {
+      return offer.paytype === PayType.SALARY ? offer.pay : offer.pay * 52 * 40;
+    });
+
+  const avgSalary =
+    salaries.length > 1
+      ? salaries.reduce((a, b) => a + b, 0) / salaries.length
+      : salaries.length === 1
+      ? salaries[0]
+      : 0;
+
+  return (
+    <div>
+      <b>${avgSalary.toFixed(2)}</b> (${(avgSalary / 52 / 40).toFixed(2)}/hr)
+      <Progress
+        value={avgSalary}
+        min={Math.min(...salaries)}
+        max={Math.max(...salaries)}
+      />
```

Comment:
It might be interesting to have some sort of expiry/trend tracking, especially if we maintain this service for a number of years, though this would be a separate feature.

Reply:
On a similar thread, a filter system for locations and positions that also impacts this track would be neat.

Reply:
I was going to have a filter modal to filter how far back to trend against, as well toggling including hours/salary in view/calculation